outputs.tf

```diff
@@ -25,6 +36,12 @@ output "resource_id" {
   value       = azurerm_cognitive_account.this.id
 }
 
+output "secondary_access_key" {
+  description = "The endpoint used to connect to the Cognitive Service Account."

```

Comment:
This output seems like an access key to me rather than an endpoint, typo?

Reply:
it a typo. Thanks!